notebooks/visualization_03_landscape.md

```diff
@@ -41,6 +41,7 @@ To run the large data set section, additional package installations are required
 
 ```
 conda install -c conda-forge datashader holoviews
+conda install -c pyviz geoviews
```

Comment:
This seems an oversight (or old habit) in their README, as geoviews can also just be installed from conda-forge